api/v1/backup_webhook.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1
 
 import (
+	"github.com/cloudnative-pg/machinery/pkg/log"
```

Comment:
🤔 maybe we should update golangci-lint's config to group these below too?

Reply:
I don't know. The last group is for imports from the same repo. The `machinery` is part of the same project but in a different repo.